array.c

```diff
@@ -8158,7 +8158,7 @@ rb_ary_deconstruct(VALUE ary)
  *      %i[foo bar baz] # => [:foo, :bar, :baz]
  *      %i[1 % *]       # => [:"1", :%, :*]
  *
- *  - \Method Kernel#Array:
```

Comment:
I think `Method` should be an exception. We don't usually deal with `Method` objects so we shouldn't link to it.

Reply:
Fixed.  Also, noted in doc guide.